src/one_dragon/base/cv_process/cv_code_generator.py

```
@@ -0,0 +1,94 @@
+                kernel_size = step.params.get('kernel_size')
+                iterations = step.params.get('iterations')
+                line = f"mask = cv2.erode(mask, np.ones(({kernel_size}, {kernel_size}), np.uint8), iterations={iterations})"
+            elif isinstance(step, CvDilateStep):
+                kernel_size = step.params.get('kernel_size')
+                iterations = step.params.get('iterations')
+                line = f"mask = cv2.dilate(mask, np.ones(({kernel_size}, {kernel_size}), np.uint8), iterations={iterations})"
+            elif isinstance(step, CvMorphologyExStep):
+                op_name = step.params.get('op')
+                cv2_op = step.op_map.get(op_name)
+                kernel_size = step.params.get('kernel_size')
+                # 在代码中直接使用cv2的枚举值，并附上注释
```

Ruff (0.14.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)